dashboard/cron.py

```diff
@@ -303,11 +303,8 @@ def update_with_bq_access(self):
             logger.debug(resource_access_df)
 
             # Because we're pulling all the data down into one query we need to manipulate it a little bit
-            # Make a copy of the access dataframe
-            resource_df = resource_access_df.copy(deep=True)
-            # Drop out the columns user and access time from resource data frame
-            resource_df = resource_df.drop(["user_id", "access_time"], axis=1)
-            # Drop out the duplicates
+            # This creates a new resource df of the resource_access with specific columns
+            resource_df = resource_access_df.filter(["resource_id", "resource_type", "name"], axis=1)
```

Comment:
Instead of `axis=1`, use `columns=` with the list of column names.  It makes the code easier to read and maintain.  No one would need to remember or look up that axis 1 refers to columns.
```suggestion
            resource_df = resource_access_df.filter(columns=["resource_id", "resource_type", "name"])
```

Reply:
I'm not [seeing columns](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.filter.html) as a kwarg for filter. I do see `axis='columns'` which is clearer. Is this what you might have meant?

Reply:
It also looks like the default for axis here is 'columns' for DataFrame so it's probably not necessary.

Reply:
I admit I didn't look up the docs for `filter()`, but `drop()` supports the `columns=` argument, so I assumed `filter()` would, too.

If the default axis is columns, then it makes sense to drop it.  If you try `columns=` with with `filter()` and it works, then I'd use that.  When it comes to working with pandas, I remember @ssciolla's comment to me in a past PR that it's better to not be so clever that it becomes complicated for future maintainers.  So I avoided chaining too many methods together and I think using named arguments makes them easier to understand.

Sorry for soapboxing. 😆

Reply:
Yeah, this is throwing an error

`TypeError: filter() got an unexpected keyword argument 'columns'`
Without axis looks like it works. I also hit #1025 when testing this and assigned that to you, should be an easy fix. I just worked around it with the ignore parameter but thinking about it, it seems like it would be worth printing a warning in the logs.